pages/linux/swapon.md

```diff
@@ -0,0 +1,23 @@
+# swapon
+
+> Enables device or file for paging or swapping.
```

Comment:
This line seems to be picked directly from the man page. I have one small suggestion - "paging" is a general concept. "swapping" is an implementation of paging to disk. We should probably just use "swapping" here for simplicity.

Reply:
Thanks for suggestion !